### PR TITLE
UI screens for branch details, project group listing and add project

### PR DESCRIPTION
**Pull request by shilpaGowda**
Created HTML skeleton for the below listed pages

- Branch listing
- Faculty listing
- Students listing
- Add project
- list project groups

### CODE REVIEW

File: app/branch/index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                  <small>Individual projects</small>
+                  <h3 class="h3">100</h3>
+                </div>
+
+                <div class="text-left">
+                  <small>Group projects</small>
+                  <h3 class="h3">100</h3>
+                </div>
+              </div>
+              <a href="details.html" class="btn btn-sm btn-link">View Details</a>
```
Comment (varunnotkar59) [inline app/branch/index.html]: Add a prevailing /

File: app/branch/index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+</head>
+
+<body>
+  <div class="container">
+    <section class="mt-5">
+      <h1 class="text-uppercase h2">Branches</h1>
+      <div class="row">
+        <div class="col-md-4 card">
+            <img src="https://loremflickr.com/cache/resized/5515_9355429249_7e51750214_n_320_240_nofilter.jpg"
+              class="card-img-top rounded-circle w-25 mt-3 mx-auto" alt="">
```
Comment (varunnotkar59) [inline app/branch/index.html]: Not sure why we are using `w-25` class here. If this is being used to centre align the image please use `d-block`

File: app/branch/students.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  <script src="scripts/modernizr.js"></script>
+  <!-- endbuild -->
+</head>
+
+<body>
+  <div class="container">
+    <section class="mt-5">
+      <h1 class="text-uppercase h2">Students - ISE</h1>
+      <div class="row mt-4">
+        <div class="col-md-3">
```
Comment (varunnotkar59) [inline app/branch/students.html]: can use `nav` tag instead of `div`

File: app/branch/students.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  <link rel="apple-touch-icon" href="apple-touch-icon.png">
+  <!-- Place favicon.ico in the root directory -->
+  <!-- build:css styles/vendor.css -->
+  <link rel="stylesheet" href="/node_modules/bootstrap/dist/css/bootstrap.min.css" type="text/css" />
+  <!-- endbuild -->
+  <!-- build:css fontawesome.css -->
+  <link rel="stylesheet" href="/node_modules/@fortawesome/fontawesome-free/css/all.min.css" type="text/css" />
+  <!-- endbuild -->
+  <!-- build:css styles/main.css -->
+  <link rel="stylesheet" href="styles/main.css">
```
Comment (varunnotkar59) [inline app/branch/students.html]: Refer to the comments in 'app/branch/students.html'

File: app/project/add.html [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  <link rel="stylesheet" href="styles/main.css">
+  <!-- endbuild -->
+  <!-- build:js scripts/modernizr.js -->
+  <script src="scripts/modernizr.js"></script>
+  <!-- endbuild -->
+</head>
+
+<body>
+  <div class="container">
+    <section class="mt-5">
```
Comment (varunnotkar59) [inline app/project/add.html]: Please use the comment from `app/branch/students.html`

File: app/project/add.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+      <h1 class="text-uppercase h2">Projects</h1>
+      <div class="row">
+        <div class="col-8">
+        <form method="POST" action="">
+          <!-- Project title -->
+          <div class="row">
+          <div class="form-group col-md-6">
```
Comment (varunnotkar59) [inline app/project/add.html]: Fix code formatting

File: app/project/add.html [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+        <!-- Project abstract -->
+          <div class="form-group mt-1">
+            <label for="description">Abstract</label>
+            <textarea rows="3" class="form-control" id="description" placeholder=""></textarea>
+          </div>
+
+          <!-- Project description -->
+          <div class="form-group mt-4">
+            <label for="description">Description</label>
+            <textarea rows="3" class="form-control" id="description" placeholder="Describe your project"
```
Comment (varunnotkar59) [inline app/project/add.html]: Increase the number of rows for description to 6

File: app/project/add.html [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+                <option value="" disabled>Select Type</option>
+                <option value="IEEE">Individual Projects</option>
+                <option value="Springer">Group projects</option>
+              </select>
+          </div>
+        </div>
+
+        <!-- Project abstract -->
+          <div class="form-group mt-1">
+            <label for="description">Abstract</label>
```
Comment (varunnotkar59) [inline app/project/add.html]: Change for attribute to *abstract*

File: app/project/group.html [hunk 9]
```diff
@@ -0,0 +1,305 @@
...
+      <di class="row">
+        <div class="col-md-6">
+
+          <div class="project-card d-flex justify-content-between py-2">
+            <div class="description">
+              <h5 class="text-primary">Group 1</h5>
+              <p>Project not allocated - <a href="#" data-toggle="modal" data-target="#projectModal">Allocate
+                  project</a></p>
+              <div class="meta d-flex justify-content-between align-items-center">
+                <div>
```
Comment (varunnotkar59) [inline app/project/group.html]: This tag is not required

File: app/project/group.html [hunk 10]
```diff
@@ -0,0 +1,305 @@
...
+        </div>
+
+        <div class="col-md-6">
+          <div class="project-card d-flex justify-content-between py-2">
+
+            <div class="description">
+              <h5 class="text-primary">Group 2</h5>
+              <div>Project - <a href="#">Handwriting Recognition </a></div>
+              <div class="meta d-flex justify-content-between align-items-center">
+                <div class="mt-3">
```
Comment (varunnotkar59) [inline app/project/group.html]: This tag is not required.

File: app/project/group.html [hunk 11]
```diff
@@ -0,0 +1,305 @@
...
+                  <img src="http://lorempixel.com/50/50/people/" alt="Team member" data-toggle="tooltip"
+                    data-placement="top" title="Kruthi" class="team-member rounded-circle shadow-sm">
+                  <img src="http://lorempixel.com/50/50/people/" alt="Team member" data-toggle="tooltip"
+                    data-placement="top" title="Kruthi" class="team-member rounded-circle shadow-sm">
+                </div>
+                <!-- <div>Guide alloted - <a href="#">Varun Umesh </a></div> -->
+              </div>
+            </div>
+
+            <div class="actions ml-5">
```
Comment (varunnotkar59) [inline app/project/group.html]: Please remove `.ml-5`

File: app/project/group.html [hunk 12]
```diff
@@ -0,0 +1,305 @@
...
+                      data-placement="top" title="Kruthi" class="team-member rounded-circle shadow-sm">
+                    <img src="http://lorempixel.com/50/50/people/" alt="Team member" data-toggle="tooltip"
+                      data-placement="top" title="Kruthi" class="team-member rounded-circle shadow-sm">
+                  </div>
+                  <!-- <div>Guide alloted - <a href="#">Varun Umesh </a></div> -->
+                </div>
+              </div>
+
+              <div class="actions ml-5">
+                <button class="btn btn-light btn-circle shadow-sm mt-3 ml-2" data-toggle="tooltip" data-placement="top"
```
Comment (varunnotkar59) [inline app/project/group.html]: `ml-2` class not required here

File: app/project/group.html [hunk 13]
```diff
@@ -0,0 +1,305 @@
...
+                    <img src="http://lorempixel.com/50/50/people/" alt="Team member" data-toggle="tooltip"
+                      data-placement="top" title="Kruthi" class="team-member rounded-circle shadow-sm">
+                    <img src="http://lorempixel.com/50/50/people/" alt="Team member" data-toggle="tooltip"
+                      data-placement="top" title="Kruthi" class="team-member rounded-circle shadow-sm">
+                  </div>
+                  <!-- <div>Guide alloted - <a href="#">Varun Umesh </a></div> -->
+                </div>
+              </div>
+
+              <div class="actions ml-5">
```
Comment (varunnotkar59) [inline app/project/group.html]: Please remove `.ml-5`

File: app/project/group.html [hunk 14]
```diff
@@ -0,0 +1,305 @@
...
+                <button class="btn btn-light btn-circle shadow-sm mt-3 ml-2" data-toggle="tooltip" data-placement="top"
```
Comment (varunnotkar59) [inline app/project/group.html]: Please remove `.ml-2`

File: app/project/group.html [hunk 15]
```diff
@@ -0,0 +1,305 @@
...
+                <option>Project 3</option>
+                <option>Project 4</option>
+                <option>Project 5</option>
+              </select>
+            </div>
+          </form>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-outline-secondary" data-dismiss="modal">Close</button>
+          <button type="button" class="btn btn-primary">Save changes</button>
```
Comment (varunnotkar59) [inline app/project/group.html]: type should be Submit

File: app/styles/main.scss [hunk 16]
```diff
@@ -5,83 +5,6 @@
...
-  /* Space out the masthead */
-  .header {
-    margin-bottom: 2rem;
-  }
-  /* Remove the bottom border on the jumbotron for visual effect */
-  .jumbotron {
-    border-bottom: 0;
-  }
+.container {
+  width: 64rem;
```
Comment (varunnotkar59) [inline app/styles/main.scss]: Please change this to `max-width`

File: app/branch/details.html [hunk 17]
```diff
@@ -0,0 +1,76 @@
...
+    <section class="mt-5">
+      <h1 class="text-uppercase h2">Branch Details - ISE</h1>
+      <div class="row mt-4">
+        <div class="col-md-3">
+            <ul class="list-group">
+                <li class="list-group-item active">
+                  <a href="" class="text-white">Details</a>
+                </li>
+                <li class="list-group-item">
+                    <a href="faculty.html" class="text-dark">Faculty</a>
```
[the remark this answers is not in the document]
Comment (shilpaGowda) [reply app/branch/details.html]: Adding / redirecting to http://localhost:9000/details.html which is a wrong path.

**Pull request merged**